shared/src/lib/dialects/index.ts

```diff
@@ -5,6 +5,14 @@ import { PostgresData } from "./postgresql";
 import { SqliteData } from "./sqlite";
 import { SqlServerData } from "./sqlserver";
 
+// Fix (part 3 of 3) Issue #1399 - int64s not displaying properly
+// Fixes error 'BigInt' is not defined in lib/db/clients/sqlite.js
+declare global {
+  interface BigIntConstructor {
+    toJSON:()=>BigInt;
+  }
+}
+
```

Comment:
actually, it seems like this `declare global` TS code should be moved into that new `/initializers/` file, too

Reply:
(reviewing myself... 🙄)